kata-webhook/create-certs.sh

```diff
@@ -20,8 +20,12 @@ kubectl create secret generic \
     --from-file=cert.pem=./webhook.crt \
     --dry-run -o yaml > ./deploy/webhook-certs.yaml
 
+BASE64_OPT="-w0"
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
`OSTYPE` ? who defined it?

Reply:
@devimc `OSTYPE` is part of `bash`.  You can read more about it here. https://www.gnu.org/software/bash/manual/html_node/Bash-Variables.html

Reply:
:exploding_head: thanks